tests/test_parser.py

```diff
@@ -0,0 +1,278 @@
+from typing import Any
+
```

Comment:
The parser is mainly tested by integration tests, while the number of unit tests is limited. Could the grammar tests of RecordFlux be reused here?

Reply:
Good idea. Done in 355a6a3.